bot/bot.lua

```diff
@@ -41,7 +41,7 @@ function on_binlog_replay_end()
 end
 
 function msg_valid(msg)
-  -- Dont process outgoing messages
+  -- Don't process outgoing messages
```

Comment:
Not sure if apostrophes `'` need to be escaped, since I don't know how Lua comments work:
```
--[[
<code>
]]--
```
Is all I know.